catroid/src/main/AndroidManifest.xml

```diff
@@ -243,6 +243,11 @@
         <service android:name=".transfers.MediaDownloadService" />
         <service android:name="com.parrot.freeflight.service.DroneControlService" />
         <service android:name=".utils.StatusBarNotificationManager$NotificationActionService" />
-
-    </application>
+        <service
+            android:name=".ui.cloudmessaging.CustomFirebaseMessagingService">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT"/>
+            </intent-filter>
+        </service>
```

Comment:
Well, there are ways to subscribe and unsubscribe to the topic but not to the service itself (not available in its API). Though I can search for disabling the service (if there is one available).